integration/vm_factory/vm_templating_test.sh

```diff
@@ -54,9 +54,21 @@ check_vm_template_factory() {
 	[ $res -eq 1 ] || die "template factory is not set up, missing state file"
 }
 
+clean_vm_template() {
+	res=$(mount | grep ${template_tmpfs_path} | wc -l)
+	if [ $res -ne 0 ]; then
+		echo "cleaning vm template"
+		umount ${template_tmpfs_path}
+		[ $? -eq 0 ] || die "clean vm template failed"
```

Comment:
> Since this script is run with `set -e`, I don't think we'll ever see that message. You'd have to do something like:
> 
> ```shell
> { umount ${template_tmpfs_path; ret=$?; } || true
> [ $ret -eq 0 ] || die "clean vm template failed"
> ```
> 
> The `res=$(...)` works about because the final command in the pipeline (`wc`) always works.

Thanks! Post a new version according to your comments.